plugins/source/azure/resources/services/compute/skus.go

```diff
@@ -19,21 +19,27 @@ func SKUs() *schema.Table {
 		Description: "https://learn.microsoft.com/en-us/rest/api/compute/resource-skus/list?tabs=HTTP#resourceskusresult",
 		Multiplex:   client.SubscriptionMultiplexRegisteredNamespace("azure_compute_skus", client.Namespacemicrosoft_compute),
 		Transform: transformers.TransformWithStruct(&armcompute.ResourceSKU{},
-			transformers.WithPrimaryKeys("Name"),
+			transformers.WithPrimaryKeys("Name", "ResourceType"),
 		),
 		Columns: schema.ColumnList{
 			client.SubscriptionIDPK,
 			schema.Column{
-				Name:            "_sku_hash",
+				Name:            "locations_hash",
 				Type:            schema.TypeString,
-				Resolver:        calcEntryHash,
+				Resolver:        calcLocationsHash,
+				CreationOptions: schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+			schema.Column{
+				Name:            "capabilities_hash",
+				Type:            schema.TypeString,
+				Resolver:        calcCapabilitiesHash,
```

Comment:
Is there documentation that justifies this change? And if this is about performance can we benchmark the memory savings and or CPU savings?

Reply:
No, I just had to inspect the `az vm list-skus` output.
Basically, there are a lot of duplicates around locations and different capabilities. However, this `jq` transformation lets us see the same amount of unique lines:
```bash
az vm list-skus \
  jq -c '.[] | {name, resourceType, locations, capabilities} | (.locations |= sort) | if .capabilities then . else . + {capabilities:[]} end | (.capabilities? |= sort)' \
  sort | wc -l
```

So, this is totally empirical change.

Reply:
I don't think we should be introducing breaking changes unless there is a specific benefit for users, and unless we can benchmark the memory improvements I don't think this change rises to that level

Reply:
The work to investigate memory issues on Azure is still on going, see https://github.com/cloudquery/cloudquery/issues/9269.

So far removing `hashstructure` did not show any memory or performance improvements in my internal tests, so I'd put this PR hold for now.

We might want to remove `hashstructure` for other reasons to do it (1. and 3. described in https://github.com/cloudquery/cloudquery/issues/9391#issue-1637475685), but it shouldn't be tied to a specific table.